core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -354,6 +400,16 @@ public void configureProperties() throws IOException {
   public abstract OutputStream create(String path) throws IOException;
 
   /**
+   * Creates a file in the under file system with the indicated name and permission status.
+   *
+   * @param path The file name
+   * @param ps the permission status to set for the file
+   * @return A {@code OutputStream} object
+   * @throws IOException if a non-Alluxio error occurs
+   */
+  public abstract OutputStream create(String path, PermissionStatus ps) throws IOException;
```

Comment:
I think at this point it would make sense to switch to provide two create methods:

`public abstract OutputStream create(String path)`

and

`public abstract OutputStream(String path, CreateOptions options)`

where options contains, permission status, block size (in bytes), and replication